test/Microsoft.AspNetCore.Diagnostics.HealthChecks.Tests/HealthCheckMiddlewareTests.cs

```diff
@@ -322,9 +322,9 @@ public async Task CanSuppressCacheHeaders()
             var builder = new WebHostBuilder()
                 .Configure(app =>
                 {
-                    app.UseHealthChecks("/health", new HealthCheckOptions()
+                    app.UseHealthChecks("/health", new HealthCheckMiddlewareOptions()
                     {
-                        SuppressCacheHeaders = true,
+                        AllowCachingResponses = true,
```

Comment:
`AllowCachingResponses  = false`?

Reply:
no, this is correct. It prevents the cache headers from being added. The property was renamed, but it's purpose and default value are the same.